src/backend/gl/src/device.rs

```diff
@@ -624,12 +624,15 @@ impl d::Device<B> for Device {
         ID: IntoIterator,
         ID::Item: Borrow<pass::SubpassDependency>,
     {
+        let max_attachments = self.share.context.get_parameter_i32(glow::MAX_COLOR_ATTACHMENTS) as usize;
```

Comment:
we should have this in the limits, so this line should be re-using the limits instead of re-querying it